File: programacao/python/respostas/selecao/32.py
anos = int(input("Digite a idade humana: "))

if anos < 0:
    print("Erro")
elif anos <= 2:
    print(f"{anos * 10.5:.1f} anos caninos")
else:
    print(f"{21 + (anos - 2) * 4} anos caninos")

File: programacao/c/respostas/selecao/32.c
#include <stdio.h>

int main(void) {
    int anos;
    float caninos;

    printf("Digite a idade humana: ");
    scanf("%d", &anos);

    if (anos < 0) {
        printf("Erro\n");
    } else if (anos <= 2) {
        caninos = anos * 10.5;
        printf("%.1f anos caninos\n", caninos);
    } else {
        printf("%d anos caninos\n", 21 + (anos - 2) * 4);
    }

    return 0;
}
